.github/workflows/deploy-gcp-tests.yml

```diff
@@ -211,22 +215,22 @@ jobs:
         id: get-disk-name
         run: |
           LOCAL_STATE_VERSION=$(grep -oE "DATABASE_FORMAT_VERSION: .* [0-9]+" "$GITHUB_WORKSPACE/zebra-state/src/constants.rs" | grep -oE "[0-9]+" | tail -n1)
-          echo "LOCAL_STATE_VERSION: $LOCAL_STATE_VERSION"
+          echo "STATE_VERSION: $LOCAL_STATE_VERSION"
 
           # Try to find an image generated from the main branch
           # Fields are listed in the "Create image from state disk" step
           #
           # TODO: require ${NETWORK} in the name after PR #4391 merges to main, and runs a full sync
           #       network should replace [a-z]*
-          CACHED_DISK_NAME=$(gcloud compute images list --filter="name~${{ inputs.disk_prefix }}-main-[0-9a-f]+-v${LOCAL_STATE_VERSION}-[a-z]*-${{ inputs.disk_suffix }}" --format="value(NAME)" --sort-by=~creationTimestamp --limit=1)
+          CACHED_DISK_NAME=$(gcloud compute images list --filter="name~${{ inputs.zebra_state_dir || inputs.disk_prefix }}-main-[0-9a-f]+-v${LOCAL_STATE_VERSION}-[a-z]*-${{ inputs.disk_suffix }}" --format="value(NAME)" --sort-by=~creationTimestamp --limit=1)
           echo "main Disk: $CACHED_DISK_NAME"
 
           if [[ -z "$CACHED_DISK_NAME" ]]; then
               # Try to find an image generated from any other branch
               #
               # TODO: require ${NETWORK} in the name after PRs #4391 and #4385 merge to main
               #       network should replace [a-z]*
-              CACHED_DISK_NAME=$(gcloud compute images list --filter="name~${{ inputs.disk_prefix }}-.+-[0-9a-f]+-v${LOCAL_STATE_VERSION}-[a-z]*-${{ inputs.disk_suffix }}" --format="value(NAME)" --sort-by=~creationTimestamp --limit=1)
+              CACHED_DISK_NAME=$(gcloud compute images list --filter="name~${{ inputs.zebra_state_dir || inputs.disk_prefix }}-.+-[0-9a-f]+-v${LOCAL_STATE_VERSION}-[a-z]*-${{ inputs.disk_suffix }}" --format="value(NAME)" --sort-by=~creationTimestamp --limit=1)
```

Comment:
I'm not sure I understand these changes:
 - why are they needed?
 - what do the resulting disk names look like?
 - do we also need to change the workflow that creates the disks?

They seem complicated, but they don't have any documentation.

Reply:
Is this a weird hack to deal with the input and output disk prefixes?

Because it might be better to just split them, like I suggested:
https://github.com/ZcashFoundation/zebra/pull/4268/files#r870909774

Reply:
This makes more sense after looking at https://github.com/ZcashFoundation/zebra/pull/4269/files#r871981831

I'm ok with it, but I'd like some documentation, so we don't forget how it works.

Reply:
Explained with [docs(ci): explain disk mounting logic](https://github.com/ZcashFoundation/zebra/pull/4268/commits/bdb217b3b8b760e834c537fe81ad39dc40bf4f2c)

Reply:
I'm using `inputs.zebra_state_dir` as this is the name of the `cached state` directory for `zebra`, but also the name of the input disk. The same happens in #4269 where the input disk is based on `inputs.lwd_state_dir` and this is the name of the `cached state` directory for `lightwalletd` and also for the input disk.

We might be able to use better names, but I'd prefer a different PR for naming this variables better.

`disk_prefix` will exclusively be used to name the image, and for tests that do not use a `zebra_state_dir` to work, but builds a cached state.

As tests have discrepancy between used directories, this was needed to reduce the amount of variables being added to this workflow, and extra steps.

Reply:
I agree that this PR isn't the right time to refactor the input variables.

Can you please document how these variables are used, so it's easier to fix future bugs, or do future refactors?

Reply:
Documented with  [docs(ci): add a description for confusing input names](https://github.com/ZcashFoundation/zebra/pull/4268/commits/769f92b25d4a6e6b2937413c8342b3604219300f)